terraform/gen/referencedocs.go.tpl

```diff
@@ -1,3 +1,14 @@
+{/* Content generated by teleport-plugins/terraform/gen/main.go DO NOT EDIT */}
+---
+title: Terraform provider resources
+description: Terraform provider resources reference
+---
+
```

Comment:
```suggestion
---
title: Terraform provider resources
description: Terraform provider resources reference
---
{/* Content generated by teleport-plugins/terraform/gen/main.go DO NOT EDIT */}
```
Otherwise the title won't render